src/Microsoft.AspNetCore.Server.Kestrel.Core/KestrelServerOptions.cs

```diff
@@ -36,6 +37,11 @@ public class KestrelServerOptions
         public SchedulingMode ApplicationSchedulingMode { get; set; } = SchedulingMode.Default;
 
         /// <summary>
+        /// Gets or sets a value that controls whether synchronous IO is allowed for the <see cref="HttpContext.Request"/> and <see cref="HttpContext.Response"/> 
```

Comment:
That makes sense. At the same time, we want to reference HttpContext in the doc comment, right? I'll just leave it as is then.